src/utils/webRequestUtil.js

```diff
@@ -6,18 +6,27 @@
 import { SERVER_END_POINT } from "../configs/server";
 import axios from "axios";
 
-const basicRequest = (type, { url, headers, body, origin }) => {
-  const config = {
-    method: type,
-    url: origin ? origin : SERVER_END_POINT + url,
-    headers: {
-      Accept: "application/json",
-      "Content-Type": "application/json",
-      ...headers,
-    },
-    data: body,
-  };
-
+const basicRequest = (type, { url, headers, body, origin, data }) => {
```

Comment:
webRequestUtil을 수정했는데, 기존 다른 axios에는 영향이 없나요?

Reply:
```suggestion
const basicRequest = (type, { url, headers, body, origin, **data** }) => {
```
data가 있는 경우에만 조건문 존재, Pdf Upload 시에만 data post로 넘김.
기존 다른 axios 영향 X